esx_service/utils/auth_data.py

```diff
@@ -44,6 +47,34 @@
 
 UPGRADE_README = "https://github.com/vmware/docker-volume-vsphere/blob/master/docs/misc/UpgradeFrom_Pre0.11.1.md"
 
+
+class DBMode(object):
```

Comment:
python does not have enum :-) , besides class provide nice extras like string conversions...